.github/workflows/build-test-package.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       max-parallel: 2
       matrix:
-        python-version: ["3.6", "3.7", "3.8", "3.9"]
+        python-version: ["3.6", "3.7", "3.9"]
```

Comment:
Is there are reason we test on 3.6 and 3.7 at this point?  Should we instead test on something more modern?

Reply:
I just pushed a commit to use versions 3.8-3.12, which is 3.8 plus the [current tensorflow specifications](https://www.tensorflow.org/install/pip).  I also moved the `pip install --upgrade typing-extensions` command in `build-test-package.yml` to be after the tensorflow install that occurs with the histomics_stream install command, `pip install .[tensorflow,torch,zarr]`.  If Python 3.8 fails, I will remove it again.

Reply:
Torch doesn't like Python 3.12, so I've changed the range to by Python 3.8-3.11.